Modules/Scripted/ShapeAnalysisToolBox/Resources/DataImporter.ui

```diff
@@ -229,6 +486,78 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QLabel" name="label_2">
```

Comment:
I this empty a label added by error. If not it may be sensible to give it a name